src/views/ProviderView/index.js

```diff
@@ -179,9 +201,13 @@ module.exports = class ProviderView {
     } catch (err) {
       // Nothing, restriction errors handled in Core
     }
-    if (!isCheckbox) {
-      this.donePicking()
-    }
+  }
+
+  removeFile (id) {
+    const { currentSelection } = this.plugin.getPluginState()
+    this.plugin.setPluginState({
+      currentSelection: currentSelection.filter((file) => file.id !== id)
+    })
```

Comment:
Does this do anything, or is it here in case we might need it? Current solution doesn’t allow removing files from ProviderView, correct?